CHANGES.md

```diff
@@ -1,5 +1,7 @@
 
 ## Changelog
+### 1.8.0
```

Comment:
Does this version bump need to be reflected somewhere?

Reply:
This won't actually bump the version (unfortunately), but I wanted to record the changes that are coming with the next version. I understand there are a number of changes on master since the last release as well, so we should at least start a new section. I will add an issue to update the CHANGELOG with everything that would be bundled with a new release.